setup.cfg

```diff
@@ -40,19 +40,23 @@ packages = find:
 install_requires =
   aacgmv2
   apexpy
+  geospacepy
   numpy
-  OMMBV
+  OMMBV>=1.0
   pandas
   pysat>=3.0
   pyEphem
-  sgp4
+  sgp4>=2.7
```

Comment:
These are not supposed to have version limits, see many previous times discussions about this.